style.css

```diff
@@ -398,3 +398,110 @@ span {
   background: #fff;
   margin-bottom: 30px;
 }
+
+.desktop-tools {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  .nav-link {
+    display: none;
+  }
+
+  .nav-container {
+    padding: 10px 40px;
+  }
+
+  .desktop-tools {
+    display: inline;
+  }
+
+  .display-tools,
+  li,
+  a {
+    padding-right: 10px;
+    height: 0%;
+  }
+
+  .major-container {
+    padding: 40px 50px;
+  }
+
```

Comment:
- Hey, please, let's adjust our styling to improve the look of this part of your portfolio by enhancing the background image to be close to the Figma design. See the photos below:


*your design*
![Screenshot from 2022-12-16 17-01-00](https://user-images.githubusercontent.com/59397159/208139457-b86b0e5c-5202-4fa0-b997-e86e7a31e04c.png)

*Figma design*
![Screenshot from 2022-12-16 17-00-41](https://user-images.githubusercontent.com/59397159/208139470-40a09e18-edaf-4985-9485-e4589d8d1771.png)